browser/resources/settings/brave_default_extensions_page/brave_default_extensions_page.html

```diff
@@ -85,13 +85,19 @@
         sub-label="$i18n{hangoutsEnabledDesc}"
         on-settings-boolean-control-change="onHangoutsEnabledChange_">
     </settings-toggle-button>
-    <template is="dom-if" if="{{ unstoppableDomainsEnabled_ }}">
+    <template is="dom-if" if="{{ decentralizedDnsEnabled_ }}">
       <div class="settings-box">
         <div class="start">$i18n{resolveUnstoppableDomainsDesc}</div>
         <settings-dropdown-menu id="unstoppableDomainsResolveMethodType"
                                 pref="{{prefs.brave.unstoppable_domains.resolve_method}}"
                                 menu-options="[[unstoppableDomainsResolveMethod_]]">
       </div>
+      <div class="settings-box">
+        <div class="start">$i18n{resolveENSDesc}</div>
+        <settings-dropdown-menu id="ensResolveMethodType"
+                                pref="{{prefs.brave.ens.resolve_method}}"
+                                menu-options="[[ensResolveMethod_]]">
+      </div>
```

Comment:
In a separate PR perhaps, but I wonder if these should go under the wallet settings. I know it's not really wallet, but it's along the same lines for people that care about these options.  @jamesmudgett thoughts?

Reply:
https://github.com/brave/brave-browser/issues/14516